src/backend/src/app.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+import os
+import json
+import logging
+from json import JSONDecodeError
+import pii_redacter
+from fastapi import FastAPI, HTTPException
+from fastapi.concurrency import asynccontextmanager
+from fastapi.staticfiles import StaticFiles
+from fastapi.responses import JSONResponse, FileResponse
+from pydantic import BaseModel
+from semantic_kernel_orchestrator import SemanticKernelOrchestrator
+from azure.identity.aio import DefaultAzureCredential
+from semantic_kernel.agents import AzureAIAgent
+from utils import get_azure_credential
+from aoai_client import AOAIClient, get_prompt
+
+from azure.search.documents import SearchClient
+
+# Run locally with `uvicorn app:app --reload --host 127.0.0.1 --port 7000`
+# Comment out for local testing:
+# from dotenv import load_dotenv
+# load_dotenv()
+
+# Environment variables
+PROJECT_ENDPOINT = os.environ.get("AGENTS_PROJECT_ENDPOINT")
+MODEL_NAME = os.environ.get("AOAI_DEPLOYMENT")
+CONFIG_DIR = os.environ.get("CONFIG_DIR", ".")
+config_file = os.path.join(CONFIG_DIR, "config.json")
+
+# Read config.json file from the config directory
+if os.path.exists(config_file):
+    with open(config_file, "r") as f:
+        AGENT_IDS = json.load(f)
+else:
+    AGENT_IDS = {}
+
+# Comment out for local testing:
+# AGENT_IDS = {
+#     "TRIAGE_AGENT_ID": os.environ.get("TRIAGE_AGENT_ID"),
+#     "HEAD_SUPPORT_AGENT_ID": os.environ.get("HEAD_SUPPORT_AGENT_ID"),
+#     "ORDER_STATUS_AGENT_ID": os.environ.get("ORDER_STATUS_AGENT_ID"),
+#     "ORDER_CANCEL_AGENT_ID": os.environ.get("ORDER_CANCEL_AGENT_ID"),
+#     "ORDER_REFUND_AGENT_ID": os.environ.get("ORDER_REFUND_AGENT_ID"),
+# }
+
```

Comment:
imo - we should throw here if we see AGENT_IDS is not populated, otherwise you will get a very hard to debug error

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'asd'
```

looking error - these are super annoying to debug